libs/components/forms/src/lib/modules/radio/radio.component.ts

```diff
@@ -215,55 +232,61 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
   @Output()
   // eslint-disable-next-line @angular-eslint/no-output-native
   public get change(): Observable<SkyRadioChange> {
-    return this._change;
+    return this.#_change;
   }
 
   /**
    * Fires when the selected value changes.
    */
   @Output()
   public get checkedChange(): Observable<boolean> {
-    return this._checkedChange;
+    return this.#_checkedChange;
   }
 
   /**
    * Fires when the selected value changes.
    */
   @Output()
   public get disabledChange(): Observable<boolean> {
-    return this._disabledChange;
-  }
-
-  public get inputId(): string {
-    return `sky-radio-${this.id}-input`;
+    return this.#_disabledChange;
```

Comment:
☝️ All of these outputs should be using the `asObservable` pattern described above.

Reply:
Fixed